FastSurferCNN/run_prediction.py

```diff
@@ -304,7 +304,7 @@ def handle_cuda_memory_exception(exception: RuntimeError, exit_on_out_of_memory:
                         help="Directory in which evaluation results should be written. "
                              "Will be created if it does not exist. Optional if full path is defined for --pred_name.")
     parser.add_argument('--hires', action="store_true", default=False, dest='hires',
-                        help="Switch on hires processing (no conforming to 1mm, but to smallest voxel size.")
+                        help="Switch on hires processing (no conforming to 1mm, but to smallest voxel size).")
```

Comment:
Description for hires flag is missing in run_fastsurfer.sh. Should be added there as well.